app/views/posts/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_with model: [@user, @post], url: user_posts_path(@user) do |form| %>
+  <div class="form-title field">
```

Comment:
- [ ] As required for the project and to make your application be completed according to the required expectations, kindly ensure there is a form in the `views/comments/new.html.erb` file which accepts input for creating comments as right it is missing. *And you should ensure that user can create a comment using the comment form you added**. Here is the error received when user wants to create a new comment:
![Screenshot from 2023-06-16 06-55-00](https://github.com/shasherazi/blog/assets/83042742/0187cabd-e773-4a36-9230-bf8bf94a51f5)